core/src/main/scala/org/bitcoins/core/protocol/ln/LightningNetworkPrefix.scala

```diff
@@ -0,0 +1,36 @@
+package org.bitcoins.core.protocol.ln
+
+import org.bitcoins.core.config.{ MainNet, NetworkParameters, RegTest, TestNet3 }
+
+sealed abstract class LightningNetworkParams {
+  def value: String
+  def network: NetworkParameters
+}
+
+object LightningNetworkPrefix {
```

Comment:
I think this should be renamed to `LIghtningNetworkParams` -- also we need to decide if want to consistently call things `Ln` (abbreviation) or `LightningNetwork` -- i'm in favor of brevity.

Reply:
If you are in favor of brevity, we shoud rename all of this class to `LnParams`, `LnPrefix` etc

Reply:
I agree, `LnParams` and `LnPrefix` are better.